tdrs-backend/tdpservice/users/views.py

```diff
@@ -51,6 +51,18 @@ def set_profile(self, request, pk=None):
         )
         return Response(serializer.data)
 
+    @action(methods=["PATCH"], detail=False)
+    def request_access(self, request, pk=None):
```

Comment:
@raftmsohani added this new request_access endpoint for the `access_request` field to be toggled from the frontend.